source/statement.c

```diff
@@ -134,3 +135,27 @@ void candl_statement_usr_cleanup(osl_statement_p statement) {
     free(stmt_usr);
   }
 }
+
+candl_statement_usr_p candl_statement_usr_clone(candl_statement_usr_p usr) {
```

Comment:
Doxygen comments missing. It's the same for the next function `candl_statement_find_label`.